js/commonMenu.js

```diff
@@ -314,90 +306,3 @@ module.exports.aboutBraveMenuItem = () => {
     }
   }
 }
-
-module.exports.buildBraveryMenu = function (settings, init) {
```

Comment:
See the commit comment, the global options are coming in preferences instead later today.